src/main/java/com/epam/ta/reportportal/ws/converter/builders/PatternTemplateBuilder.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.ta.reportportal.ws.converter.builders;
+
+import com.epam.ta.reportportal.commons.validation.Suppliers;
+import com.epam.ta.reportportal.entity.pattern.PatternTemplate;
+import com.epam.ta.reportportal.entity.pattern.PatternTemplateType;
+import com.epam.ta.reportportal.exception.ReportPortalException;
+import com.epam.ta.reportportal.ws.model.ErrorType;
+import com.epam.ta.reportportal.ws.model.project.config.pattern.CreatePatternTemplateRQ;
+
+import java.util.function.Supplier;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+public class PatternTemplateBuilder implements Supplier<PatternTemplate> {
+
+	private PatternTemplate patternTemplate;
+
+	public PatternTemplateBuilder(CreatePatternTemplateRQ createPatternTemplateRQ) {
```

Comment:
move it to the method and create empty constructor.

Reply:
done